events/codepipeline_test.go

```diff
@@ -18,7 +18,7 @@ func TestCodePipeLineEventMarshaling(t *testing.T) {
 		t.Errorf("could not open test file. details: %v", err)
 	}
 
-	// de-serialize into CognitoEvent
```

Comment:
Thanks for noticing this, but it's best if the PR we merge is focused only on the new feature. Feel free to open a separate PR that bundles up the docs and comment fixes you've found.

Reply:
I have cherry-picked the other changes to a separate branch and will shortly push those changes